### ISSUE TITLE
houdini search doesn't work some letters

### ISSUE THREAD

**Issue by bkingery** [byu-animation/byu-animation-tools#51]
These include: j, k, u, w, y, z

This might be an issue to take up with sidefx

**Comment by garretthoyos** [comment]
I can contact Seth and ask him and I can post about it on SideFX's website. Is that the way we should go about this?

**Comment by bkingery** [comment]
Yeah.  I want to try to understand what is going on a little better to make sure it's not something that we are doing...
I don't think it is though.  Ethan might know a little more.

**Comment by eestrada** [comment]
Be sure to check the Houdini journals. We generally are a few dailies build behind the latest build. I have sent in bug reports before that were actually fixed in newer versions of Houdini. Check under Support->Journals on www.sidefx.com.

**Comment by bkingery** [comment]
I couldn't find anything in the Journals.  I wan to make sure it's not a preference we have set like "Show Only Opertors with Preferred Versions" or something before we take it up with sidefx.

**Comment by eestrada** [comment]
I know that Houdini creates and caches a bunch of config files on first run. This may be part of that cache. Dan mentioned it may be that they only create tokens for certain letters in the Alphabet one first run.

**Comment by ebrayton** [comment]
K, this might sound weird, and I'm still investigating it, but it doesn't seem to recognize any lower case letter when said letter is in the middle of the asset's name. Example from Chasm: _Chasm easel_ & _Chasm Engine_.... Typing E will show the _Engine_, but not _easel_ until EA is typed. 
Since, in Owned, there are no current tab menu options that have capital J in their name, it won't let you continue typing JE to get to jeff.

**Comment by mrmurphy** [comment]
That's super odd. Have we found anyone else with this issue? It seems significant..

**Comment by ebrayton** [comment]
I submitted a bug report to SideFX. I'll let you know if I hear anything from them, but we might want to just go to Title Case assets.

**Comment by bkingery** [comment]
I'm changing this to low priority as we are just waiting on Houdnini now.

**Comment by bkingery** [comment]
This is out of our hands for the moment.  I'm closing the issue.
